src/modules/CVs/components/skills/skills.tsx

```diff
@@ -0,0 +1,119 @@
+import { useEffect, useMemo, useState } from "react";
+import { Navigate, useParams } from "react-router-dom";
+import { Mastery, SkillMastery } from "cv-graphql";
+import { routes } from "@/router";
+import { useTranslation } from "react-i18next";
+import { useAuthUser } from "@/services/auth-service";
+import { useCvById } from "@/modules/CVs/api";
+import { SkillsWrapper } from "@/components/skills-wrapper";
+import { PlusOutlined } from "@ant-design/icons";
+import { useAddSkill } from "@/modules/CVs/components/skills-dialog/skills-dialog.tsx";
+import { AddSkillSchemaType } from "src/modules/CVs/components/skills-dialog/schemas";
+import { useAddCvSkill } from "@/api/add-cv-skill-mutation";
+import { useSkills } from "@/api/get-skills-query";
+import { useUpdateCvSkill } from "@/api/update-cv-skill-mutation";
+import { Button, Flex, Spin } from "antd";
+
+const Skills = () => {
+    const { cvId } = useParams<{ cvId: string }>();
+    const currentUser = useAuthUser();
+    if (!cvId || !currentUser) {
```

Comment:
early return (and if you don't want to/can't return after running all the logic below, you could actually create some simple page components/wrappers where you would do this simple check and just pass actually existing & valid `cvId` as a prop for this component) (`currentUser.email` could also be replaced with `currentUser?.email` without this check at all (since similar authentication check happens at `main-layout` level anyway))